src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForMultipleBuildingBlocks.cs

```diff
@@ -0,0 +1,134 @@
+      {
+         private readonly IInteractionTasksForChildren<Module, EventGroupBuildingBlock> _interactionTasksForApplicationMoleculeBuilder;
+         private readonly IInteractionTasksForChildren<Module, InitialConditionsBuildingBlock> _interactionTasksForInitialCondition;
+         private readonly IInteractionTasksForChildren<Module, MoleculeBuildingBlock> _interactionTasksForMolecule;
+         private readonly IInteractionTasksForChildren<Module, PassiveTransportBuildingBlock> _interactionTasksForPassiveTransport;
+         private readonly IInteractionTasksForChildren<Module, ParameterValuesBuildingBlock> _interactionTasksForParameterValues;
+         private readonly IInteractionTasksForChildren<Module, MoBiReactionBuildingBlock> _interactionTasksForMobiReaction;
+         private readonly IInteractionTasksForChildren<Module, MoBiSpatialStructure> _interactionTasksForMobiSpatialStructure;
+         private readonly IInteractionTasksForChildren<Module, ObserverBuildingBlock> _interactionTasksForObserver;
+         private readonly IInteractionTaskContext _interactionTaskContext;
+
+         public InteractionTasksForMultipleBuildingBlocks(
+            IInteractionTasksForChildren<Module, EventGroupBuildingBlock> interactionTasksForApplicationMoleculeBuilder,
+            IInteractionTasksForChildren<Module, InitialConditionsBuildingBlock> interactionTasksForInitialCondition,
+            IInteractionTasksForChildren<Module, MoleculeBuildingBlock> interactionTasksForMolecule,
+            IInteractionTasksForChildren<Module, PassiveTransportBuildingBlock> interactionTasksForPassiveTransport,
+            IInteractionTasksForChildren<Module, ParameterValuesBuildingBlock> interactionTasksForParameterValues,
+            IInteractionTasksForChildren<Module, MoBiReactionBuildingBlock> interactionTasksForMobiReaction,
+            IInteractionTasksForChildren<Module, MoBiSpatialStructure> interactionTasksForMobiSpatialStructure,
+            IInteractionTasksForChildren<Module, ObserverBuildingBlock> interactionTasksForObserver,
+            IInteractionTaskContext interactionTaskContext)
+         {
+            _interactionTasksForApplicationMoleculeBuilder = interactionTasksForApplicationMoleculeBuilder;
+            _interactionTasksForInitialCondition = interactionTasksForInitialCondition;
+            _interactionTasksForMolecule = interactionTasksForMolecule;
+            _interactionTasksForPassiveTransport = interactionTasksForPassiveTransport;
+            _interactionTasksForParameterValues = interactionTasksForParameterValues;
+            _interactionTasksForMobiReaction = interactionTasksForMobiReaction;
+            _interactionTasksForMobiSpatialStructure = interactionTasksForMobiSpatialStructure;
+            _interactionTasksForObserver = interactionTasksForObserver;
+            _interactionTaskContext = interactionTaskContext;
```

Comment:
I would expect all these names to end with `...BuildingBlock`. There are interaction tasks for builders, but in this case we aren't going to use them.